README.md

```diff
@@ -37,6 +38,13 @@ This template comes with:
 8. `rails s`
 9. You can now try your REST services!
 
+## How to use with docker
+
+1. Have docker and docker-compose installed
+2. build and run the application with `docker-compose up`
```

Comment:
build and run are actually two commands, `docker-compose build && docker compose up`

Reply:
I know, but if the image has not been built, `docker-compose up` builds it, and then runs it.